emcc.py

```diff
@@ -2791,6 +2791,58 @@ def in_directory(root, child):
   return False
 
 
+def parse_string_value(text):
```

Comment:
these two internal methods could be nested inside the parent function `parse_value`